src/datareaders/lucid/lucid_parser.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime
+import pandas as pd
+import math
+from src.datareaders.resources import get_data_resource
+from src.datareaders.table_enumerations import Sources
+from src.datareaders.data_object_holders import Point, PointType, PointValue
+
+
+class LucidParser:
+    """ A class for parsing the Lucid CSVs.
+        I don't see this being used once we get this live,
+        but for now it's what we are using. """
+
+    def __init__(self):
+        self.data = None  # Pandas DataFrame for storing CSV
+        self.point_identities = None
+        self.point_values = None
+
+    def read_csv(self, file_name):
+        self.data = pd.read_csv(file_name, dtype=object)
+
+    def create_point_identities(self):
+        """
+        Function to create the point names for the Points database.
+        Iterates through the column headers row 4 of Lucid CSV to parse out names with units.
+
+        :return: None, sets the class variable to the list of new points.
+        """
+        point_identities = [] # This will be a list of each of the column headers as a Point object.
+
+        point_names = list(self.data.iloc[3].copy())  # Get row that includes all point name data.
+        for i in range(1, len(point_names)):
+            # Get the name, the building name (sometimes the same thing) and the units information
+            name, building_name, description = point_names[i].split(" - ")
+
+            name = building_name + " - " + description.split("(")[0]
+
+            if "Old Meter" in description:  # We need to differentiate between old meters and new ones.
```

Comment:
Are old meters still in use and still reporting? And if so, then how would this handle if a currently active meter gets replaced and its description gets changed to have old meter in it? Just wondering how we're handling that sort of stuff, or if we're just not worrying about it for now

Reply:
That's a good reminder to think about what might happen when data changes. I think we aren't worrying about it for now.